woocommerce/woocommerce-paypal-payments
# Add a new namespaced script loader (3771)

### Description

This PR attempts to remedy various issues stemming from multiple PayPal scripts conflicting.

The new loader, `PayPalScriptLoading.js`, requires passing a namespace when loading an instance of the PayPal script. It also tries to separate the processing of the config options and performing the actual script loading.

It also migrates the following to the new loader:
- Axo and Axo Block
- Express payment buttons
- Google Pay

### Steps to test
1. Test Fastlane in the Classic and Block checkouts. Ensure the script loads correctly and doesn't cause conflicts with the smart buttons. (reload the page a couple of times)
2. Ensure Google Pay loads correctly in all contexts. 

### Screenshots

N/A
